config/routes.rb

```diff
@@ -1,13 +1,14 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
+  devise_for :users
```

Comment:
Make sure the routes are nested within the `users` resources, we need the recipes to be dependent on the user who signs in

Reply:
Thanks for that